.github/workflows/test.sh

```diff
@@ -29,7 +29,7 @@ else
     if [ "${1}" = "pyqt5" ]; then
         pip install pyqt5==${PYQT5_VERSION:-"5.15"}.* PyQtWebEngine==${PYQT5_VERSION:-"5.15"}.*
     elif [ "${1}" = "pyqt6" ]; then
-        pip install pyqt6==${PYQT6_VERSION:-"6.2"}.* PyQt6-WebEngine==${PYQT6_VERSION:-"6.2"}.*
+        pip install pyqt6==${PYQT6_VERSION:-"6.3"}.* PyQt6-WebEngine==${PYQT6_VERSION:-"6.3"}.*
```

Comment:
Now that its a separate package, shouldn't we now do

```suggestion
        pip install pyqt6==${PYQT6_VERSION:-"6.3"}.* PyQt6-WebEngine==${PYQT6_VERSION:-"6.3"}.* PyQt6-Qt6==${PYQT6_QT_VERSION:-"6.3"}.*
```

so it can be overridden on individual matrix jobs, like the other bindings can?

It might be good to keep at least one matrix job on the "LTS" version...though with with the policy changes on LTS for FOSS, I'm not sure.

Reply:
About the LTS version maybe we could add it sure, but then we will need to have a validation to change the pip line to see if it needs to include or not `PyQt6-Qt6==${PYQT6_QT_VERSION:-"6.3"}.*`, right?